engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelSchemaValidationError.kt

```diff
@@ -321,6 +323,28 @@ sealed interface NadelSchemaValidationError {
         override val subject = overallField
     }
 
+    data class IncompatibleHydrationArgumentType(
+            val parentType: NadelServiceSchemaElement,
+            val overallField: GraphQLFieldDefinition,
+            val remoteArg: RemoteArgumentDefinition,
+            val outputFieldType: GraphQLOutputType,
+            val actorArgInputType: GraphQLInputType,
+    ) : NadelSchemaValidationError {
+        val service: Service get() = parentType.service
+
+        override val message = run {
+            val hydrationArgName = remoteArg.name
+            val of = makeFieldCoordinates(parentType.overall.name, overallField.name)
+            val uf = "${parentType.underlying.name}.${remoteArg.remoteArgumentSource.pathToField?.joinToString(separator = ".")}"
+            val s = service.name
+            "Field $of tried to hydrate with argument $hydrationArgName using value from underlying field $uf from " +
```

Comment:
Btw this is no longer underlying field.

Reply:
thanks yea will need to update the comments on the other errors as well